install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -118,6 +120,7 @@ type MetaConfiguration struct {
 }
 
 type UpgradeConfiguration struct {
+	Registry  string              `json:registry,omitempty`
```

Comment:
Simply because the configuration variables SYNDESIS_REGISTRY & UPGRADE_REGISTRY are defined separately & the latter was used explicitly in the templates. I can collapse them together if preferred?

Reply:
Wouldnt it just be easier to just have a single image field that contains the reg, image, and tag?

Reply:
Updated and collapsed 2 REGISTRY keys for now.